src/main/kotlin/org/jitsi/jicofo/bridge/BridgeConfig.kt

```diff
@@ -123,8 +123,8 @@ class BridgeConfig private constructor() {
 
     val regionGroups: Set<Set<String>> by config {
         "jicofo.bridge".from(JitsiConfig.newConfig).convertFrom<ConfigObject> {
-            val regionGroups = it["region-groups"] as? ConfigList ?: emptyList<ConfigValue>()
-            regionGroups.map { regionGroupConfigList ->
+            val regionGroupsConfigList = it["region-groups"] as? ConfigList ?: emptyList<ConfigValue>()
+            regionGroupsConfigList.map { regionGroupConfigList ->
```

Comment:
Now you have two vals called `regionGroupConfigList`.  🙂

Reply:
No, one is `regionGroup*s*ConfigList` (which is the list of region groups) and the other is `regionGroupConfigList` (which is the list of regions in the group). I tried to improve the naming a bit. I didn't intentionally make this hard to read, I promise :)